$OpenBSD: patch-src_redis_c,v 1.22 2014/11/06 09:21:28 dcoppa Exp $

The fancy ascii art looks like a mess when logging to syslog

--- src/redis.c.orig	Fri Sep 19 17:49:16 2014
+++ src/redis.c	Thu Nov  6 10:01:25 2014
@@ -3295,7 +3295,7 @@ int main(int argc, char **argv) {
     initServer();
     if (server.daemonize) createPidFile();
     redisSetProcTitle(argv[0]);
-    redisAsciiArt();
+    if (!server.syslog_enabled) redisAsciiArt();
 
     if (!server.sentinel_mode) {
         /* Things not needed when running in Sentinel mode. */
